update/src/Pos/Update/Logic/Local.hs

```diff
@@ -79,7 +79,7 @@ getMemPool
     :: (MonadReader ctx m, HasLens UpdateContext ctx UpdateContext, MonadIO m)
     => m MemPool
 getMemPool = msPool <$>
-    (atomically . readTVar . mvState =<< views (lensOf @UpdateContext) ucMemState)
+    (readTVarIO . mvState =<< views (lensOf @UpdateContext) ucMemState)
```

Comment:
👍 this is cleaner, and seems OK to update in this PR, even though it's unrelated to partial field accessors.

Reply:
We can thank @erikd 's pre-commit hook for this :)

Reply:
Ah right, I forgot about HLint.